### PR TITLE
Add acct company

### PR DESCRIPTION
**Pull request by macindex**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @macindex, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 9f3899a).  See below for details:

```
acct.md: Company title "Abstract" doesn't match filename (expected ~ "abstract.md")
acct.md: Invalid section: "Company acct".  Expected one of: ["Company blurb","Company size","Remote status","Region","Company technologies","Office locations","How to apply"]
acct.md: Required section "Company blurb" not found.
acct.md: Empty section: "Remote status". Fill it in or leave it out instead.

4 problems detected
```

Please fix these issues and push a new commit to this pull request.

**Comment by auto-nylen** [comment]
Hi @macindex, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit 74b2f33). :heavy_check_mark:

**Comment by macindex** [comment]
Add acct company

File: company-profiles/acct.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+We are ACCT (ACupula Consulting & Technology), a software consultancy that creates and transforms technologies into value for high performance businesses. Today we operate in 15 countries creating technologies for the largest e-businesses in the country and the world.
+
+## Company size
+
+51-200 employees
+
+## Remote status
+
+Treinning analyst - We are ACCT (ACupula Consultoria e Tecnologia), a software consultancy that creates and transforms technologies into value for high performance businesses.
```
[the remark this answers is not in the document]
Comment (macindex) [reply company-profiles/acct.md:13]: right, I would like to correct do another commit?

[the remark this answers is not in the document]
Comment (dougaitken) [reply company-profiles/acct.md:13]: Yes please @macindex - push another commit to this branch.

**Comment by macindex** [comment]
update remote status

**Pull request merged**